src/Shiny.BluetoothLE/PairingConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shiny.BluetoothLE
```

Comment:
This should likely just being a string argument instead of a full object to the pairing method

Reply:
I think a single pin paramater will not be enough in the future.
There is many other options in bonding procedure like security level, etc or the configuration could contain callback methods to request pin from the calling application.

Reply:
I'd rather not leave it open ended.  While I'm happy to take additional PR's to update it in the future, it should be single purpose for now.

Reply:
I'll have one more BLE project in September. I think, I'll have some more ideas for this.

Reply:
ok